lasagne/nonlinearities.py

```diff
@@ -46,12 +46,10 @@ def softmax(x):
 # tanh
 def tanh(x):
     """Tanh activation function :math:`\\varphi(x) = \\tanh(x)`
-
```

Comment:
why did you remove this whitespace? It's inconsistent with the other functions now.